src/Microsoft.ML.FastTree/Application/DominationLossApplication.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace Microsoft.ML.Trainers.FastTree.Internal
+namespace Microsoft.ML.Trainers.FastTree
 {
 #if OLD_DATALOAD
```

Comment:
>OLD_DATALOAD [](start = 4, length = 12)

Is this whole file inside the #if?

Reply:
I think we could in principle safely remove this, but I'd make that a separate PR.

Reply:
remove the file? of the directive?

---
In reply to: [254797836](https://github.com/dotnet/machinelearning/pull/2442#discussion_r254797836) [](ancestors = 254797836)

Reply:
it looks like. 

---
In reply to: [254505167](https://github.com/dotnet/machinelearning/pull/2442#discussion_r254505167) [](ancestors = 254505167)